src/test/java/seedu/address/model/person/DebtTest.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class DebtTest {
+    private Debt test1 = new Debt("5");
+    private Debt test2 = new Debt("4");
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Debt((String) null));
+    }
+
+    @Test
+    public void constructor_invalidDebt_throwsIllegalArgumentException() {
+        String invalidDebtString = "1.001";
+        assertThrows(IllegalArgumentException.class, () -> new Debt(invalidDebtString));
+    }
+
+    @Test
+    public void isValidDebt() {
+        assertThrows(NullPointerException.class, () -> Debt.isValidDebt(null));
+
+        assertFalse(Debt.isValidDebt(""));
+        assertFalse(Debt.isValidDebt(" "));
+        assertFalse(Debt.isValidDebt("abc"));
+
+        assertTrue(Debt.isValidDebt("-2"));
```

Comment:
Negative debt is valid? 👀

Reply:
The debt class is to store debt within a Person. We can have negative debt to indicate the person owing money to the user instead.

The ParserUtil.parseDebt is used to parse debt passed into the ChangeDebtCommand, while Debt.isValidDebt is used to validate Debt.

Reply:
Ok! Should document that you can record negative debt in `Debt` class to signify that the user of FriendDex owes money to the person instead.